README.md

```diff
@@ -113,30 +119,38 @@ KubeSphere uses a loosely-coupled architecture that separates the [frontend](htt
 
 ## Latest release
 
-🎉 KubeSphere 3.2.0 is now available! See the [Release Notes For 3.2.0](https://kubesphere.io/docs/release/release-v320/) for the updates.
-
+🎉 KubeSphere 3.2.0 is now available! It has supported up to Kubernetes 1.22, see the [Release Notes For 3.2.0](https://kubesphere.io/docs/release/release-v320/) for the updates.
```

Comment:
It has supported up to Kubernetes 1.22, see the -> The latest Kubernetes version 1.22 is now supported. See the

Reply:
Actually, there are some known issues with v1.22, we used the word `experimental` in ks-installer https://github.com/kubesphere/ks-installer#prerequisites .